newsroom/utils.py

```diff
@@ -41,6 +45,14 @@ def get_entity_or_404(_id, resource):
     return item
 
 
+def get_file(key):
+    file = request.files.get(key)
+    if file:
+        filename = secure_filename(file.filename)
+        file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
```

Comment:
we should store the files to `app.media`, otherwise we would lost them on each deployment